src/components/Upload.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import PropTypes from 'prop-types';
+
+const Upload = ({ onDrop, children, ...props }) =>
+  (<Dropzone
+    {...props}
+    // What do we accept as valid?
+    accept="image/jpeg, image/png"
+    onDrop={onDrop}
+  >
+    {children}
+  </Dropzone>);
+
+Upload.propTypes = {
+  onDrop: PropTypes.func.isRequired,
+  children: PropTypes.node.isRequired,
+};
+
+export default Upload;
```

Comment:
This is an abstract component that can be styled as we want by overriding "style"